bot/cogs/moderation/utils.py

```diff
@@ -132,11 +132,11 @@
     log.trace(f"Sending {user} a DM about their {infr_type} infraction.")
 
     embed = discord.Embed(
-        description=textwrap.dedent(f"""
+        description=textwrap.shorten(textwrap.dedent(f"""
             **Type:** {infr_type.capitalize()}
             **Expires:** {expires_at or "N/A"}
             **Reason:** {reason or "No reason provided."}
-            """),
+            """), width=2048, placeholder="..."),
```

Comment:
This is hard to read. Split them up by assigning the dedent to a variable above.